.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2021": true
+    },
+    "extends": "eslint-config-airbnb-base",
```

Comment:
@eric-gade does this now disagree with https://github.com/18F/guides/pull/98/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R41?

Reply:
Hmm, it seems to make no difference whatsoever either way, which is confusing. Been looking into it